src/main/java/storage/DataFileReader.java

```diff
@@ -335,4 +346,29 @@ private void createNewCheatSheet(boolean isMarkedFavourite,
         newCheatSheet.setFavourite(isMarkedFavourite);
         cheatSheetList.add(newCheatSheet);
     }
+
+    /**
+     * Reads the save file line by line and passes each line to applySettings method.
+     */
```

Comment:
Perhaps you could include the required params and exceptions thrown?

Reply:
ah yikes forgot about that HAHAH. thanks